com.unity.render-pipelines.high-definition/Runtime/PostProcessing/Components/DepthOfField.cs

```diff
@@ -66,11 +84,17 @@ public sealed class DepthOfField : VolumeComponentWithQuality, IPostProcessCompo
         //
 
         /// <summary>
-        /// Sets the distance to the focus point from the Camera.
+        /// Sets the distance to the focus plane from the Camera.
```

Comment:
The distance to the focus plane from the Camera.

The API docs guidance says to avoids gets/sets - https://confluence.unity3d.com/pages/viewpage.action?spaceKey=DOCS&title=API+docs+content+and+formatting#heading-Propertiesandfields